packages/manager/modules/pci/src/projects/project/data-processing/notebooks/delete-notebook/delete-notebook.controller.js

```diff
@@ -0,0 +1,19 @@
+export default class {
+  /* @ngInject */
+  constructor(dataProcessingService) {
+    this.dataProcessingService = dataProcessingService;
+    this.deleteNotebook = this.deleteNotebook.bind(this);
+  }
+
+  deleteNotebook() {
+    return this.dataProcessingService
+      .deleteNotebook(this.projectId, this.notebookId)
+      .then(() => {
+        this.goBack();
+      });
```

Comment:
**question:** No error handling?

Reply:
Not required for alpha but might come in later PR